contracts/core/mpl-rewards/v1/test/MplRewards.t.sol

```diff
@@ -724,4 +732,77 @@ contract MplRewardsTest is DSTest {
         assertEq(rewardToken.balanceOf(address(stakers[1])),                  (dTime1_rpt + dTime2_rpt) * 30);
     }
 
+    function test_midEpochReset() external {
+        MplRewardsOwner owner      = new MplRewardsOwner();
+        SomeERC2258 rewardToken    = new SomeERC2258("RWT", "RWT");
+        SomeERC2258 stakingToken   = new SomeERC2258("SKT", "SKT");
```

Comment:
I think RT and ST make more sense here

Reply:
Yep. I'll fix across the tests. W not needed.